src/metabase/api/card.clj

```diff
@@ -468,4 +469,5 @@
   (db/select [Card :name :id], :enable_embedding true, :archived false))
 
 
-(api/define-routes)
+(api/define-routes
+  (middleware/streaming-json-response POST_:card-id_query))
```

Comment:
yeah it sort of does. I suppose you could use `metabase.api.common.internal/route-fn-name` which is the internal fn that generates the magic names

```clojure
(route-fn-name 'POST "/:card-id/query") ; -> POST_:card-id_query
```